### DISCUSSION TITLE
Pester tests: moving from v4 to v5

### DISCUSSION THREAD

**Discussion by giscus** [rajbos/rajbos.github.io#23] [Comments]
# Pester tests: moving from v4 to v5

Pester tests: moving from v4 to v5

https://devopsjournal.io/blog/2021/05/25/Moving-pester-to-version-5

**Comment by g0pinath** [comment]
Great stuff, thanks for sharing. One update this  needs $PesterConfig.TestResult.Enabled = $True

**Reply by rajbos** [reply]
Thanks! I've added it to the last example codeblock.
